application_sdk/decorators/mcp_tool.py

```diff
@@ -0,0 +1,59 @@
+"""
+MCP tool decorator for marking activities as MCP tools.
+
+This module provides the @mcp_tool decorator that developers use to mark
+activities for automatic exposure via Model Context Protocol.
+"""
+
+from typing import Optional
+
+
+def mcp_tool(
+    func=None,
+    *,
+    description: Optional[str] = None,
+    name: Optional[str] = None,
+    enabled: bool = True,
+):
+    """
+    Decorator to mark activities for MCP tool exposure.
+
+    When an activity is decorated with @mcp_tool, it will be automatically
+    discovered and exposed as an MCP tool when the application runs with
+    enable_mcp=True.
+
+    Args:
+        func: The function to decorate (when used as @mcp_tool)
+        description (Optional[str]): Description for the MCP tool.
+                                   Defaults to the function docstring.
+        name (Optional[str]): Custom name for the MCP tool.
+                            Defaults to the function name.
+        enabled (bool): Whether this tool should be enabled.
+                       Defaults to True.
+
+    Examples:
+        @mcp_tool
+        async def simple_activity(self, param: str) -> str:
+            pass
+
+        @mcp_tool(description="Fetch random GIF from Giphy API")
+        async def fetch_gif(self, search_term: str) -> str:
+            # ... activity implementation unchanged ...
```

Comment:
please add the `>>>` prefix to the lines in this section of the docstring indicating the python REPL. 

You can find an example here

- https://github.com/atlanhq/application-sdk/blob/556bda03e5e95904898a0eeecb9e81a15b90823e/application_sdk/activities/common/utils.py#L42

This should fix the failing docs action